### ISSUE TITLE
change debug class name to use `label` for glamor

### ISSUE THREAD

**Issue by kentcdodds** [paypal/glamorous#242]
[Now](https://github.com/threepointone/glamor/issues/238), if you add a `label` in your css, it'll make the class name include that. Maybe we could pass the `displayName` as the `label` by default rather than adding the displayName as a className?

What do you think @threepointone?

**Comment by kentcdodds** [comment]
Note: we'll wanna decide on something soon because this is one of only a few major breaking changes for the upcoming release and I don't want to have another for a long time (ever?)

**Comment by threepointone** [comment]
Yup, that was my thinking as well. *should* work, lemme know how it goes.

**Comment by threepointone** [comment]
would you consider this a breaking change? Why?

**Comment by kentcdodds** [comment]
Yeah, probably not technically a breaking change. Would like to allow people to update snapshots only once though ;-)

**Comment by threepointone** [comment]
I also have a change incoming within a week or so when I change the hashing algorithm :(

**Comment by kentcdodds** [comment]
Well, that's even more reason to use the jest-glamor-react so that doesn't impact users 👍

**Comment by kentcdodds** [comment]
This has been merged into `next`.
